### PR TITLE
add test for entitytypepropertymetadata

### PR DESCRIPTION
**Pull request by jokedurnez**
Testing changing entitytypePropertyMetadata.

### CODE REVIEW

File: src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt [hunk 1]
```diff
@@ -183,4 +184,36 @@ class EdmTest : MultipleAuthenticatedUsersBase() {
...
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty()
+        )
+        edmApi.updateEntityTypePropertyMetadata( et.id , pt1.id, update )
+        var metadata = edmApi.getEntityTypePropertyMetadata(et.id, pt1.id);
+        Assert.assertEquals(
+                "New Title !",
```
Comment (pckeyes) [inline src/test/java/com/openlattice/rehearsal/edm/EdmTest.kt]: This is nit-picky, but I'd probably make this a private val rather than a raw string

**Review by pckeyes — DISMISSED** [review]
One small comment but otherwise looks good!

**Comment by jokedurnez** [comment]
IDK why this is showing up as "review dismissed" which sounds a bit aggressive, lol.

**Review by you-leee — APPROVED** [review]

**Comment by makosblade** [comment]
> IDK why this is showing up as "review dismissed" which sounds a bit aggressive, lol.

Daaaaaannngg, talk about DisHub 😨

**Review by makosblade — APPROVED** [review]

**Review by katherinebernstein — APPROVED** [review]

**Pull request merged**